frameworks/Rust/hyper/src/main.rs

```diff
@@ -33,8 +32,9 @@ impl Service for TechEmpower {
                 Response::new()
                     .with_header(ContentLength(HELLOWORLD.len() as u64))
                     .with_header(ContentType(Mime(TopLevel::Text, SubLevel::Plain, vec![(Attr::Charset, Value::Utf8)])))
+                    .with_header(header::Server("hyper/async".to_owned()))
```

Comment:
standard rust style is `to_string` not `to_owned` (but they do the exact same thing)